src/locales/en/translation.json

```diff
@@ -231,6 +231,7 @@
     "donationDetail4": "1 x 12-inch custom designed lantern by a local artist to take home with you ",
     "donationDetail5": "An invitation to the Winter Solstice Celebration and lighting ceremony in December",
     "goal1":"Permanent Light fixtures and lanterns on Mott St from Canal to Bayard St",
-    "goal2":"Permanent Light fixtures on Bayard St from Mott Street to Bowery St"
+    "goal2":"Permanent Light fixtures on Bayard St from Mott Street to Bowery St",
+    "partnerThanks": "Thank you to our Chinatown Partners who make this possible!"
```

Comment:
be sure also to fill in the chinese version

Reply:
how do we go about requesting translations? My chinese is awful

Reply:
Just fill the translation.cn file with the english copy for now, usually translation is done after